app/components/member/Edit/index.js

```diff
@@ -53,22 +50,21 @@ class MemberEdit extends React.PureComponent {
           }).isRequired
         ).isRequired
       }).isRequired
-    })
+    }),
+    relay: PropTypes.object.isRequired
```

Comment:
Explicitly states you want access to `this.props.relay` so ESLint doesn’t scold you (it’s since moved to `member/Edit/memberships/index.js`, FYI!)